README.md

```diff
@@ -277,15 +278,21 @@ If `L` and `R` are neither equivalent, nor subsets, but there is overlapping
 membership i.e. the intersection is not the empty set
 
 ```
-members(L) ∩ members(R) ≠ ∅ 
+members(L) ∩ members(R) ≠ ∅
 ```
 
-then any member in `forkWitness(L,R)` SHOULD create a new epoch directly 
-succeeding `L`, excluding all peers that were excluded in `R`. See figure 5.
+and the symmetric difference is not the empty set
+
+```
+members(L) △ members(R) ≠ ∅
+```
```

Comment:
We need BOTH "intersection not empty" and "symmetric difference not empty" because the intersection alone is not enough.

For instance, if `A ⊂ B` then `A ∩ B ≠ ∅` is true, but in rule 4.6. we don't want `A ⊂ B` cases, we only want "actual overlapping".